multi_host_migration/tests/cfg/multi_host.cfg

```diff
@@ -278,8 +277,31 @@
                         - dynamic_cpu_throttling:
                             sub_type = before_migrate_capability
                             sub_test = "auto-converge enable"
+                            bg_stress_test = "stressapptest -M 150 -s 600 >/dev/null 2>&1 &"
                             need_stress = yes
```

Comment:
Agree with @CongLi , you can set the param to upper level with 'yes' as default value, so that all 'yes' case can use this default value and only the 'no yes' need to be set separately.